src/js/containers/state/visualizations/awardBreakdown/AwardBreakdownContainer.jsx

```diff
@@ -88,10 +94,11 @@ export class AwardBreakdownContainer extends React.Component {
     }
 
     parseData(results) {
-    // Sum all amounts in the returned award types
+        const amountType = this.props.toggleState ? "total_outlays" : "amount";
+        // Sum all amounts in the returned award types
         const totalAmount = reduce(
             results,
-            (sum, awardType) => sum + parseFloat(awardType.amount),
+            (sum, awardType) => sum + parseFloat(awardType[amountType]),
```

Comment:
Some states have award types that are returning null for total outlays.  Arizona is one example.  I will ask the backend about this.  In the meantime we can fix it on the frontend.  When this happens the tooltip shows -- for the percentage.  The nulls are getting converted to NaN when we use parseFloat.   I don't see a reason why parseFloat is necessary here.  Let me know if you see a compelling reason to have it here.  Otherwise, removing the parseFloat fixes the problem.  

![Screenshot 2024-09-09 at 3 33 59 PM](https://github.com/user-attachments/assets/827fe48a-3230-4c4e-9093-9461950092ea)

Reply:
UPDATE:  the backend is fixing the nulls in total_outlays.  We should leave the parseFloat since it's been there forever